### PR TITLE
Deterministic example util

### PR DESCRIPTION
**Pull request by Thomaash**
This greatly reduces differences between examples' screenshots after they've been regenerated. This is a complement to the fixed layout seed included in #140.

### CODE REVIEW

File: lib/network/modules/LayoutEngine.js [hunk 1]
```diff
@@ -34,7 +34,6 @@ import TimSort from 'timsort';
 import util from 'vis-util';
 import NetworkUtil from '../NetworkUtil';
 import { HorizontalStrategy, VerticalStrategy } from './components/DirectionStrategy.js';
-import { fillLevelsByDirection } from './layout-engine'
```
Comment (mojoaxel) [inline lib/network/modules/LayoutEngine.js]: To be honest I would have expected to see any changes to the actual lib here. Why are these changes needed to fix the examples overview?

File: lib/network/modules/LayoutEngine.js [hunk 2]
```diff
@@ -1500,10 +1499,45 @@ class LayoutEngine {
    * @private
    */
   _determineLevelsDirected() {
-    this.hierarchical.levels = fillLevelsByDirection(
-      this.body.nodeIndices.map(id => this.body.nodes[id]),
-      this.hierarchical.levels
-    );
+    let minLevel = 10000;
```
Comment (mojoaxel) [inline lib/network/modules/LayoutEngine.js]: magic number. Why 10000?

**Comment by Thomaash** [comment]
Hmm, this is weird. I changed 1 line and regenerated the examples.

**Comment by Thomaash** [comment]
@mojoaxel this is what I wanted to do :smile:.
